sentry_dramatiq/__init__.py

```diff
@@ -108,7 +101,23 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
             message._scope_manager.__exit__(None, None, None)
 
 
-def _make_message_event_processor(message, integration):
+class SentryProxyMiddleware(SentryMiddleware):
+    def _has_sentry_dramatiq_integration(self):
+        # type: () -> bool
+        hub = Hub.current
+        integration = hub.get_integration(DramatiqIntegration)
+        return integration is not None
```

Comment:
These three lines is exactly the same as what you'll find at the beginning of `SentryMiddleware.before_process_message` and `SentryMiddleware.after_process_message` so I'm not sure I understand the necessity of the proxy?

Reply:
I just decouple, without this I have some problems when I want to use sentry middleware in some brokers without monkey patch, the sentry integration automatically inserts middleware at all brokers, using proxy we have complete decouple of middleware and integration behavior, without dependencie between them.

Reply:
And move the code to new method is just to prevent repeated code.

Reply:
Okay so I guess what fixes it in your case is checking if the integration is active in `before_process_message` and `after_process_message`. You should just be able to add that check to the existing methods in `SentryMiddleware`, instead of subclassing. Do that and provide a unit test showing that it works, and I'll happily merge 😊